### DISCUSSION TITLE
ScanPeriod is not working if not available in logback.xml file

### DISCUSSION THREAD

**Discussion by skumarjh-tibco** [qos-ch/logback#762] [Q&A]
Hello experts, I found the issue related to scanPeriod where it says that by default, 60 seconds is added for this but I have tried this with the latest released logback jars and if I do not add the scanPeriod attribute to the logback file, it doesn't work. 
I can see this was added in https://github.com/qos-ch/logback/commit/d99254343#diff-be7810a89cf023bdb0eda2ac5e6c12f7f61299053fb3ce5630eeb607aca23efe but in the latest release, It doesn't work. 
Can someone please help me with my confusion?

Thank you!

**Comment by skumarjh-tibco** [comment]
Solved

**Comment by ceki** [comment]
@skumarjh You may want to share your experience as to help other users?
